mantidproject/mantid
# New Algorithm Categories Page in Documentation

This PR (referring to issue #22254) contains work to build a page which indexes the algorithm categories. The Algorithm link in the documentation now goes to the index of categories, rather than to the full list of all algorithms.

There is a block of introductory text on the Algorithm category index to explain what the page is, how categories are decided and approximately what one should expect to find in the documentation of an algorithm.

There is still a link to go to the list of all algorithms if desired.

**To test:**

Build Mantid. Build the html docs:

In `build/docs/` run : `python runsphinx_html.py`

* Check that the documentation builds.
* Check that the `Algorithms` link takes you to an index of algorithm categories.
* Check the intro text on `Algorithm Index` page.
* Check that linking from the `Algorithm Index` page works properly.
* Check that no other documentation has been affected e.g. `concepts/index.html`
 
Resolves #22254 

**Release Notes** 


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
